Source/WebCore/css/CSSStyleDeclaration.idl

```diff
@@ -40,6 +40,5 @@ typedef USVString CSSOMString;
 
     // Non-standard.
     DOMString? getPropertyShorthand(optional DOMString propertyName);
-    boolean isPropertyImplicit(optional DOMString propertyName);
```

Comment:
Is this change Web compatible??? It seems plausible that there is a bunch of iOS specific content that's relying on this. A quick search on GitHub yields about 155 entries:
https://github.com/search?q=isPropertyImplicit(%20language%3AJavaScript&type=code

Reply:
Looking at the entries, they're either:
- WebInspector instance in WebKit forks
- Chrome DevTools forks (repos named devtools-frontend or inspector)
- Epiphany
- Old forks of Chromium that still have WebKit

Seems safe to remove to me?

Reply:
Perhaps @karlcow wants to take a look.